### ISSUE TITLE
Being able to add basic markdown to the text layers

### ISSUE THREAD

**Issue by riverole** [paperize/paperize#7]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to emphasize a specific part of the text (like a title) with bold. Right now with Paperize, I need to split these into two text layers, in the case where there's a clear separation of the texts. In the case where the bold text is interpolated in the middle of the text, I cannot do this.

**Describe the solution you'd like**
I would like to be able to add some basic markdown to the text (in the google sheet), for example (and probably only) bold and italic. Something like <b>whatnot<b>yeah yeah.

**Describe alternatives you've considered**
The alternative solutions right now are to split this into different text layers.

**Comment by lorennorman** [comment]
Yes! We definitely need to expose some shorthand that can be used directly in the spreadsheet data. I think we'll have to work backwards from `jsPDF`, whatever it exposes is what we have to generate of course.

I think we should experiment with the `.html` or `.fromHtml` stuff it exposes, as that would give us a lot of power to assist with a number of different features.

**Comment by Itangalo** [comment]
I too have thought about the ability to use formatted text. There is some basic formatting possible directly in Google sheets, but using markdown (or something like that) makes more sense than importing formatting used in the sheet.

**Comment by Itangalo** [comment]
Revisiting this issue after using Paperize quite massively for a short time. I can make most things work by tweaking data in spreadsheets, but not formatting text.

I'll try to find jsPDF in the source code (assuming it's a library) and see if I can find anything useful.

**Comment by Itangalo** [comment]
I'm not certain, but it seems that jsPDF nowadays can take markdown as input. (I see things indicating that it uses a markdown 0.5.0 parser when searching the repository for 'markdown'.) In best case it could work just updating the fork used for Paperize, but I have no idea where or how Paperize and jsPDF interact. And it is fair to assume that a lot of other things have changed with jsPDF as well, of course...

**Comment by Itangalo** [comment]
If markdown could be added to Paperize, I vote for text automatically being treated as markdown with an option to turn markdown parsing off.

**Comment by Itangalo** [comment]
For reference – this is where I saw markdown stuff in jsPDF:

* https://github.com/parallax/jsPDF/blob/7315ccbb54fdf47a3498cfb3c9f757da5c450242/bower.json#L62 (Looks promising)
* https://github.com/parallax/jsPDF/blob/7315ccbb54fdf47a3498cfb3c9f757da5c450242/package.json (basically same as above)
* https://github.com/parallax/jsPDF/blob/30880b41475c587dc3a0182f980883395ea638e7/jsdoc.json#L10 (lists the tag 'deprecated'...)

I don't have skills enough to say what this means, but to me it looks like a markdown parsing library is now used by jsPDF.
